library/std/src/time.rs

```diff
@@ -458,10 +458,12 @@ impl SystemTime {
     /// # Examples
     ///
     /// ```
```

Comment:
Since this is adding a sleep, this should probably be no_run.

Reply:
Maybe replace the literal sleep with `// ... time passes ...` or something.

Reply:
I would be uneasy about that since `SystemTime` is not monotonic. I don't know what the risk level is, most systems should work correctly nowadays. But I have seen various troubles with clocks on VMs that I would be hesitant to do that (since it could panic).